options/options.css

```
@@ -1,267 +1,432 @@
+h3 {
+    text-decoration: none;
+    margin-top: 1.5em;
+}
+
+code {
+    padding: 2px 4px;
+    border-radius: 4px;
+    font-size: 90%;
+    color: var(--code-text);
+    background-color: var(--code-bg);
+    font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, monospace;
```

Stylelint (17.9.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)